GoogleSignIn/Sources/GIDSignIn.m

```diff
@@ -996,24 +997,19 @@ - (void)assertValidPresentingViewController {
 }
 
 #if TARGET_OS_IOS && !TARGET_OS_MACCATALYST
-// Asserts the requested scopes are valid.
 - (void)assertValidScopes:(NSArray<NSString *> *)scopes {
-  NSDictionary *scopeToClassMapping = @{
-    kAccountDetailTypeAgeOver18Scope : [GIDVerifyAccountDetail class],
-  };
-  NSMutableString *errorMessage =
+  GIDRestrictedScopesRegistry *registry = [[GIDRestrictedScopesRegistry alloc] init];
```

Comment:
Probably better to make this a private property on `GIDSignIn`.

Reply:
Done.